src/calendar/header/index.js

```diff
@@ -151,36 +150,40 @@ class CalendarHeader extends Component {
 
     if (!this.props.hideArrows) {
       leftArrow = (
-        <TouchableOpacity
-          onPress={this.onPressLeft}
-          disabled={this.props.disableArrowLeft}
-          style={this.style.arrow}
-          hitSlop={{left: 20, right: 20, top: 20, bottom: 20}}
-          testID={testID ? `${CHANGE_MONTH_LEFT_ARROW}-${testID}`: CHANGE_MONTH_LEFT_ARROW}
-        >
-          {this.props.renderArrow
-            ? this.props.renderArrow('left')
-            : <Image
-              source={require('../img/previous.png')}
-              style={this.props.disableArrowLeft ? this.style.disabledArrowImage : this.style.arrowImage}
-            />}
-        </TouchableOpacity>
+        this.props.renderArrow
+          ? this.props.renderArrow('left', this.props.addMonth)
```

Comment:
This is a breaking change. You're moving the 'renderArrow' out of the Touchable area.